Formula/cppcheck.rb

```diff
@@ -16,13 +17,16 @@ class Cppcheck < Formula
   depends_on "cmake" => :build
   depends_on "python@3.9" => [:build, :test]
   depends_on "pcre"
+  depends_on "tinyxml2"
 
   uses_from_macos "libxml2"
```

Comment:
Don't these conflict?

Reply:
Not at all. `tinyxml2` provides a `tinyxml2.h` and a `libtinyxml2.dylib`. `libxml2` doesn't provide anything named `tiny`.

Reply:
I actually meant in functionality, e.g. don't these things do the same thing?

Reply:
Maybe? I'm not sure. I would guess not, since:

1. the build looks for an `xmllint` executable. This is provided by `libxml2`; and,
2. the build vendors its own `tinyxml2` if you don't provide one.

It might be that `libxml2` is only a build dependency, but it's hard to check since `uses_from_macos` doesn't do anything.